github/GitRelease.py

```diff
@@ -166,13 +166,21 @@ def zipball_url(self):
         return self._zipball_url.value
 
     def delete_release(self):
+        """
+        :calls: `DELETE /repos/:owner/:repo/releases/:release_id <https://developer.github.com/v3/repos/releases/#delete-a-release>`_
+        :rtype: bool
```

Comment:
Actually we don't need to return anything here. You can remove the return and add a `:rtype: None`

Reply:
done here: 0b2a6c30cd34356b3

Reply:
so there is a test expecting this method to return `True`. Shall I remove it?

Reply:
Of course, otherwise the tests will fail.

Reply:
ok, it was more a question if the assertion or the whole test case should be removed. I did the former now: 438b10c5935550f3